Apps/phironnet/ironnet.json

```diff
@@ -0,0 +1,273 @@
+{
+    "appid": "963c802e-8b8c-46d8-bd45-28e6c455f51e",
+    "name": "IronNet",
+    "description": "This app provides actions for interacting with IronDefense from Phantom",
+    "type": "endpoint",
+    "product_vendor": "IronNet",
+    "logo": "ironnet.png",
+    "logo_dark": "ironnet_dark.png",
+    "product_name": "IronDefense",
+    "product_version_regex": ".*",
+    "publisher": "IronNet",
+    "license": "Copyright (c) IronNet, 2019",
+    "app_version": "1.0.0",
+    "utctime_updated": "2019-06-22T00:27:05.820730Z",
+    "package_name": "phantom_ironnet",
+    "main_module": "ironnet_connector.pyc",
+    "min_phantom_version": "4.5.15425",
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "base_url": {
+            "description": "the base URL for the IronAPI",
```

Comment:
Capitalize first letters for all descriptions

Reply:
I have fixed the description capitalization in a new commit - but for some reason the PR is not reflecting this new commit yet.  Trying to figure out why.

Reply:
Fixed